mypy.ini

```diff
@@ -102,6 +102,9 @@ disallow_any_generics = true
 [mypy-homeassistant.util.unit_system]
 disallow_any_generics = true
 
+[mypy-pylint.*]
+disallow_any_generics = true
+
```

Comment:
I would remove `pylint.*` from `.strict-typing`. The `disallow_any_generics` checks is nice but not really important. Especially not for a plugin.